TerminalDocs/faq.yml

```diff
@@ -0,0 +1,94 @@
+
+      - question: What is conhost.exe?
+        answer: |
+          The Windows Console host, conhost.exe, is Windows' original command-line user experience. It also hosts Windows' command-line infrastructure and the Windows Console API server, input engine, rendering engine, user preferences, etc. A primary goal of Windows Console is to maintain backward compatibility, thus adding new features became prohibitive and let to the creation of the Windows Terminal. Learn more in the [Windows Terminal open-source repo](https://github.com/Microsoft/terminal#the-windows-console-host) and in the [WIndows Console docs](/windows/console/definitions#console-host).
+
+      - question: Can I save the layout of all of my open console windows when closing Windows Terminal and restore them when starting a new session?
+        answer: |
+          Yes! As of [Windows Terminal Preview v1.12.2922.0](https://github.com/microsoft/terminal/releases/tag/v1.12.2922.0), you can now save window pane layouts upon closing a terminal session with the [firstWindowPreference](./customize-settings/startup.md#behavior-when-starting-a-new-terminal-session-preview) global setting.
+
+      - question: What is being planned for Windows Terminal? Is there a development roadmap or feature request list that I can contribute to?
+        answer: |
+          Windows Terminal is under very active development. You can view the team's plans on the [Terminal 2.0 Roadmap](https://github.com/microsoft/terminal/blob/main/doc/terminal-v2-roadmap.md) document in the open-source terminal repo. New features go into [Windows Terminal Preview](https://aka.ms/terminal-preview) first, then typically a month after they've been in Preview, those features move into [Windows Terminal](https://aka.ms/terminal).
+
+          You can contribute feature Requests or ideas, as well as bug reports, security vulnerabilities, or documentation issues by [filing an issue in the terminal repo](https://github.com/microsoft/terminal/issues/new/choose).
+
+          ![Screenshot of filing an issue in terminal repo](./images/file-issue.png)
+
+      # - question: Is there a way to sync your terminal settings across devices?
+      #   answer: |
+      #     TBD
+
+      # - question: Can we expect a status bar on the bottom of Windows Terminal? 
+      #   answer: |
+      #     TBD
+
+      # - question: TBD
+      #   answer: |
+      #     TBD
```

Comment:
How do I make Windows Terminal my default shell?
Requires Windows 11... only via settings UI / dev settings page of Windows settings.